ports/winit/events_loop.rs

```diff
@@ -89,9 +102,21 @@ impl EventsLoop {
                         None,
                         &mut control_flow
                     );
+
+                    if ! self.init_win {
+                        self.init_win = true;
+                        callback(
+                            winit::event::Event::NewEvents(winit::event::StartCause::Init),
+                            None,
+                            &mut control_flow
+                        );
+                    }
```

Comment:
We should move this check before the loop starts, instead, since it will only be needed once.

Reply:
For that, `control_flow` must also move up with it. Shall I call it twice, preserving the one inside the loop ?

Reply:
That does look cleaner ... I'll have a go. :+1: